### PR TITLE
Support profile jql

### PR DESCRIPTION
**Pull request by hieudion**
We can support People() target or only filter by JQL. 
The brief check function could be a blocker so we should disable it and find another solution later.

### CODE REVIEW

File: lib/embulk/input/service/jql_service.rb [hunk 1]
```diff
@@ -205,16 +201,36 @@ def extract_value(record, name)
...
-            # convert from ms -> second
             if record["time"].present?
-              time = record["time"] / 1000
-              adjust_timezone(time)
+              value = record["time"]
+              if value > 0
+                time = record["time"] / 1000
+                adjust_timezone(time)
+              else
+                value
```
Comment (huylenq) [inline lib/embulk/input/service/jql_service.rb:210]: Out of curiosity, are there cases where record's time is negative?

**Review by huylenq — APPROVED** [review]
LGTM

File: lib/embulk/input/service/jql_service.rb (hunk 1 above)
Comment (hieudion) [reply lib/embulk/input/service/jql_service.rb:210]: I think no but there are some case the value is 0. 
And adjust_timezone(0) return negative result.

Comment (huylenq) [reply lib/embulk/input/service/jql_service.rb:210]: I see, good to know.

**Review by ylzen — APPROVED** [review]
Reviewed for Data Exfiltrations.

**Pull request merged**